src/auth/auth.service.ts

```diff
@@ -1,8 +1,11 @@
+import { ApiException } from './../shared/errors/api-exception';
 import {
   Injectable,
   Logger,
   OnModuleInit,
-  UnauthorizedException
+  UnauthorizedException,
```

Comment:
Codacy found an issue: ['UnauthorizedException' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5747194)